register.py

```diff
@@ -17,6 +18,10 @@ def register(mode="prod"):
     username = run("git config --global user.name")
     email = run("git config --global user.email")
 
+    if not username:
```

Comment:
```suggestion
    if not username or not email:
```
Do we need to check for an invalid email too?

Reply:
yes, yes we do